kotlin/services/cognito/src/main/Flutter_AWS_SDK/ios/RunnerTests/RunnerTests.swift

```diff
@@ -0,0 +1,12 @@
+import Flutter
+import UIKit
+import XCTest
+
+class RunnerTests: XCTestCase {
+
+  func testExample() {
```

Comment:
Should we be concerned that there are no tests here?

Reply:
Yes.